packages/trip-form/src/TripOptions/ModeRow.tsx

```diff
@@ -0,0 +1,86 @@
+import coreUtils from "@opentripplanner/core-utils";
+import React from "react";
+
+import { Modes, QueryParams } from "./types";
+import * as S from "./styled";
+import Checkbox from "./Checkbox";
+import {
+  categoryIsActive,
+  getCategoryPrimaryMode,
+  getSelectedModes
+} from "./util";
```

Comment:
Sorted in 775500dafb258ffc2fc5d431b8619ebe3f21b54f, but with the caveat that non-component imports come first (as prettier seems to like)

Reply:
Is that a prettier rule stated in this repo's `package.json`?

Reply:
I think it may be in the mastarm config, but I have so many versions of mastarm on my machine I'm really not sure where it came from. I think when we clean all our eslint configs this problem should be resolved.